pkg/cmd/admin/router/router.go

```diff
@@ -438,7 +440,8 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			return fmt.Errorf("router could not be created; you must specify a service account with --service-account")
```

Comment:
smarterclayton: I left that line in case they passed --service-account=""  I couldn't see anywhere else where that was caught.  I'll move it all into a method and make the check more clear.

bmeng: This patch changes things to default the SA to 'router'